src/_N001_Email.ino

```diff
@@ -121,7 +137,7 @@ boolean NPlugin_001_send(const NotificationSettingsStruct& notificationsettings,
 			if (!NPlugin_001_MTA(client, "", F("220 "))) break;
 			if (!NPlugin_001_MTA(client, String(F("EHLO ")) + notificationsettings.Domain, F("250 "))) break;
 			if (!NPlugin_001_Auth(client, notificationsettings.User, notificationsettings.Pass)) break;
-			if (!NPlugin_001_MTA(client, String(F("MAIL FROM:<")) + notificationsettings.Sender + ">", F("250 "))) break;
+			if (!NPlugin_001_MTA(client, String(F("MAIL FROM:<")) + address + ">", F("250 "))) break;
```

Comment:
Hmm, concatenating Strings can be done more efficient using `concat()` of `strformat()` helper functions. NB: When using the `%s` format specifier, you'll have to pass the `.c_str()` of a String variable.